plugins/cache_range_requests/cache_range_requests.cc

```diff
@@ -185,110 +186,105 @@ handle_read_request_header(TSCont txn_contp, TSEvent event, void *edata)
 void
 range_header_check(TSHttpTxn txnp, pluginconfig *const pc)
 {
-  char cache_key_url[8192] = {0};
-  char *req_url;
-  int length, url_length, cache_key_url_length;
-  txndata *txn_state;
-  TSMBuffer hdr_buf;
-  TSMLoc hdr_loc = nullptr;
-  TSMLoc loc     = nullptr;
-  TSCont txn_contp;
+  txndata *txn_state = nullptr;
```

Comment:
You could have moved the definition of txn_state to line 203.

Reply:
moved.